corelib/constant.py

```diff
@@ -39,3 +39,8 @@
 #object_detect_url = 'models/yolov3:predict'
 object_detect_url = 'models/efficientdet:predict'
 scene_detect_url = 'models/places:predict'
+# caption generation url
+caption_generation_url = 'models/lstm:predict'
+image_vectorization_url = 'models/xception:predict'
+dict_wordtoix_path = 'corelib/Caption Generator/wordtoix.pickle'
```

Comment:
all data items should stay in ./data/<name_of_model>/
check how its done for for char_dict_path, ord_map_dict_path and follow the same pattern